apps/astarte_appengine_api/lib/astarte_appengine_api/device/device.ex

```diff
@@ -755,6 +755,20 @@ defmodule Astarte.AppEngine.API.Device do
     end
   end
 
+  defp cast_value(:double, anyvalue) do
+    {:ok, anyvalue + 0.00}
```

Comment:
```suggestion
    {:ok, anyvalue + 0.0}
```
Also, add a comment to remember why we're doing this (JSON does not distinguish between ints and floats etc etc)